observability/power_monitoring/power-monitoring-release-notes.adoc

```diff
@@ -15,6 +15,38 @@ These release notes track the development of {PM-title} in the {product-title}.
 
 For an overview of the {PM-operator}, see xref:../power_monitoring/power-monitoring-overview.adoc#power-monitoring-about-power-monitoring_power-monitoring-overview[About {PM-shortname}].
 
+[id="power-monitoring-release-notes-0-2"]
+== {PM-shortname-c} 0.2 (Technology Preview)
+
```

Comment:
It seems worthwhile to state that Power Monitoring 0.2 includes kepler 0.7.10 and kepler-operator 0.12.0.

Reply:
As discussed with @sthaha, I have mentioned this information as a separate sentence below the release title as:

This release includes the following version updates:
* Kepler 0.7.10
* Power monitoring Operator 0.13.0